editions/tw5.com/tiddlers/wikitext/HTML Entities.tid

```diff
@@ -0,0 +1,60 @@
+created: 20130204165019000
+modified: 20220822133445149
+tags: WikiText
+title: HTML Entities
+type: text/vnd.tiddlywiki
+
+!! Summary
+
+Use HTML entities to enter characters that can't easily be typed on an ordinary keyboard. They take the form of an ampersand (`&`), an identifying string, and a terminating semi-colon (`;`), e.g. `&amp;`.
```

Comment:
It is clear not to use contractions like "can't", but instead to spell them out in full eg "cannot". (I've made a note to clarify the docs on this https://tiddlywiki.com/#Spelling)